packages/ui/react-ui/__tests__/CrossmintPayButton.spec.tsx

```diff
@@ -14,31 +14,83 @@ global.fetch = jest.fn(() => fetchReturns);
 // TODO(#61): make this automatically mocked in every test suite
```

Comment:
IGNORE THIS FILE.

This has its own PR. https://github.com/Crossmint/crossmint-client-sdk/pull/63